acquisitions/src/main/resources/samples/mod-orders/compositeOrders/po-listed-print-monograph.json

```diff
@@ -8,17 +7,11 @@
     "ABCDEFGHIJKLMNOPQRSTUV"
   ],
   "orderType": "One-Time",
-  "poNumber": "268758",
   "reEncumber": false,
-  "vendor": "d0fb5aa0-cdf1-11e8-a8d5-f2801f1b9fd1",
+  "vendor": "c6dace5d-4574-411e-8ba1-036102fcdc9b",
```

Comment:
**Question**
Why you changed ids?

Reply:
To match the global ids for integration tests, so that I don't need to create new vendors etc.